vendor/hal_wifi_default.te

```diff
@@ -5,3 +5,4 @@ r_dir_file(hal_wifi_default, debugfs_wlan)
 
 allow hal_wifi_default kernel:system module_request;
 allow hal_wifi_default tombstone_wifi_data_file:dir rw_dir_perms;
+allow hal_wifi_defauli tombstone_wifi_data_file:file create;
```

Comment:
`defauli`? Must be `default` with a `T` at the end.

Also, can't we replace these with an `rw_dir_file`?

Reply:
oh yeah typo ... and create is not included with rw macro

Reply:
Looks like we need to add a macro for that then. Also, since the wifi HAL wants to create a tombstone file, it most likely also wants to write to it... `create_file_perms`?

Reply:
marin sometime ago I was doing it but anyone else was refusing to use marcros ... but yeah I will update this